src/ui/Word/index.scss

```diff
@@ -6,7 +6,6 @@
 }
 
 .sendbird-word__url {
-  margin: 0 4px;
```

Comment:
With this change https://github.com/sendbird/sendbird-uikit-react/pull/961/files#r1477653792 we don't need this margin value any more.